packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -70,7 +69,7 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   }) : assert(dragStartBehavior != null),
        super(debugOwner: debugOwner, kind: kind);
 
-  static VelocityTracker _defaultBuilder(PointerEvent ev) => VelocityTracker();
+  static VelocityTracker _defaultBuilder(PointerEvent ev) => VelocityTracker(ev.kind);
```

Comment:
nit: it'd be nice if the variable was called `event` instead of just `ev`.

Reply:
Done